requirements.dev.txt

```diff
@@ -1,6 +1,71 @@
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    pip-compile requirements.dev.in
+#
+appdirs==1.4.4
+    # via black
```

Comment:
I usually recommend the option which disables the comments

Reply:
I will look into it next time.